lib/perl/Genome/Model/Tools/Speedseq/Base.t

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use above 'Genome';
+use Test::More tests => 3;
+
+my $pkg = 'Genome::Model::Tools::Speedseq::Base';
+use_ok ($pkg);
+
+my $version = 'test';
+my $expected_path = '/gscmnt/gc2719/halllab/users/cchiang/bin/speedseq';
```

Comment:
We should ask Colby to update the gsc branch on his github with his fix, and then build our own executable